vega_sim/scenario/ideal_market_maker_v2/scenario.py

```diff
@@ -198,4 +215,8 @@ def run_iteration(self, vega: VegaServiceNull, pause_at_completion: bool = False
         run_with_console=True,
         seconds_per_block=40,  # Heuristic
     ) as vega:
-        scenario.run_iteration(vega=vega, pause_at_completion=True)
+        scenario.run_iteration(
+            vega=vega, 
+            pause_at_completion=True,
+            run_with_console=False,
```

Comment:
run_with_console here I think shouldn't do anything because it's the outer VegaServiceNull which is controlling that.